lib/LedgerNodeOperations.js

```diff
@@ -46,9 +48,37 @@ module.exports = class LedgerNodeOperations {
    * @return a Promise that resolves once the operation completes.
    */
   async validate(operation) {
+    if(!(operation && typeof operation === 'object')) {
+      throw new BedrockError('Operation must be an object.', 'TypeError');
+    }
+
+    _validateOperation(operation);
+
     const {ledgerNode} = this;
     const config = await ledgerNode.config.get();
     const validatorConfigs = config.operationValidator || [];
+
     return validator.validate(operation, validatorConfigs, {ledgerNode});
   }
 };
+
+function _validateOperation(operation) {
+  const context = operation['@context'];
+  _validateContext(context);
+  const validation = validate('webledger-validator.operation', operation);
+  if(!validation.valid) {
+    throw validation.error;
+  }
+}
+
+function _validateContext(context) {
+  const wlContextUrl = constants.WEB_LEDGER_CONTEXT_V1_URL;
+  if(typeof context === 'string' && context !== wlContextUrl) {
+    throw new BedrockError(`Operation context must be "${wlContextUrl}"`,
+      'SyntaxError', {context});
```

Comment:
Style nit: If you drop any params to a function to a new line, always drop the first one too.